VerloopSDK/VLConfig.swift

```diff
@@ -363,4 +380,32 @@ extension VLConfig {
         return self.department
     }
     
+    var getNavTitle: String {
```

Comment:
I see you routed the the title change from an instance of vlconfig. I'd like to make sure that the object properties are not accessible publicly. Else we can maintain an internal instance and not part of config. We'll discuss.